client/landing/gutenboarding/onboarding-block/acquire-intent/vertical-select/index.tsx

```diff
@@ -59,109 +57,129 @@ const VerticalSelect: React.FunctionComponent< Props > = ( { onSubmit } ) => {
 	const { siteVertical, siteTitle } = useSelect( select => select( ONBOARD_STORE ).getState() );
 	const { setSiteVertical, resetSiteVertical } = useDispatch( ONBOARD_STORE );
 
-	const [ inputValue, setInputValue ] = React.useState( siteVertical?.label ?? '' );
+	const inputText = inputRef?.current?.innerText || '';
```

Comment:
No, because of this: https://github.com/Automattic/wp-calypso/pull/40645/files#diff-b5234b8c6d62cf33d02a21482433b9ceR28
We usually pass `null` as an argument but in this case an actual tag it's needed for the [imperative updates](https://github.com/Automattic/wp-calypso/pull/40645#discussion_r402186180) we're doing.